scripts/integration-test-all.sh

```diff
@@ -3,31 +3,35 @@
 # "stable" mode tests assume data is static
 # "live" mode tests assume data dynamic
 
-
-SCRIPT=`basename ${BASH_SOURCE[0]}`
+SCRIPT=$(basename ${BASH_SOURCE[0]})
 TEST=""
+QTD=1
+SLEEP_TIMEOUT=5
```

Comment:
Perhaps we should increase this, I still see some of these failing with Travis when this is 5.

Reply:
this is now configurable, check the help output with -h and makefile task.